packages/client/src/components/Modals/SavedSearchPanel.vue

```diff
@@ -45,12 +75,35 @@ export default {
   validations: {},
   watch: {},
   computed: {
-    ...mapGetters({}),
+    ...mapGetters({
+      savedSearches: 'grants/savedSearches',
+    }),
   },
   mounted() {
+    this.setup();
   },
   methods: {
-    ...mapActions({}),
+    ...mapActions({
+      createSavedSearch: 'grants/createSavedSearch',
+      updateSavedSearch: 'grants/updateSavedSearch',
+      deleteSavedSearchAPI: 'grants/deleteSavedSearch',
+      fetchSavedSearches: 'grants/fetchSavedSearches',
+      changeSelectedSearchId: 'grants/changeSelectedSearchId',
+    }),
+    setup() {
+      this.fetchSavedSearches();
+    },
+    editSavedSearch(e) {
+      this.changeSelectedSearchId(e.target.getAttribute('searchid'));
```

Comment:
@jakekreider this is what I was planning on using to communicate between the `SavedSearchPanel`, `GrantTable` and `SearchPanel`. By having a shared sense of what is the "currently applied saved search" we should be able to show the correct search filters and results.

This is not fully hooked up, but I figured you can use this to display the correct data in the search filters.